inc/options/class-wpseo-options.php

```diff
@@ -379,6 +379,24 @@ public static function reset_ms_blog( $blog_id ) {
 		}
 	}
 
+	/**
+	 * Saves the option to the database.
+	 *
+	 * @param string $wpseo_options_group_name The name for the wpseo option group in the database.
+	 * @param string $option_name              The name for the option to set.
+	 * @param *      $option_value             The value for the option.
+	 *
+	 * @return string Saving successful.
```

Comment:
I believe this returns a boolean and not a string. Also, please expand the description a bit more.